.github/workflows/sync-with-upstream.yaml

```diff
@@ -59,23 +59,23 @@ jobs:
             --exclude .git/ --exclude .github/workflows/.template_version template/ repo/
 
           # Check if the branch exists in the current repo.
-          cd repo/
-          if [ $(git rev-parse --verify origin/sync-$LATEST_VERSION 2>/dev/null) ]; then
+          cd repo/ || exit 1
+          if [ "$(git rev-parse --verify origin/sync-"""$LATEST_VERSION""" 2>/dev/null)" ]; then
             echo "Branch sync-$LATEST_VERSION already exists. Exiting..."
             exit 0
           fi
```

Comment:
Hm, so in
```bash
cd repo/ || exit 1
```
the script exits without a message and with return code `1`, and in the next line it exits with a message and return code `0`. Perhaps we can keep things a bit more consistent?

Reply:
See commit [3dd239a](https://github.com/jenstroeger/python-package-template/pull/505/commits/3dd239a6f2cec3327a94ff788e658632b6a1f5d6).